src/Jackalope/Tools/Console/Command/InitDoctrineDbalCommand.php

```diff
@@ -84,7 +84,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return self::RETURN_CODE_NO_FORCE;
         }
 
-        $schema = new RepositorySchema;
+        $schema = new RepositorySchema([], $connection);
```

Comment:
this seems to have super weird side effects in the schema class :-(
can you try to figure out what we should do there instead? looks like we abuse the $connection to decide between different situations...

Reply:
seems only be an unexpected call in the unit test. did fix that or are there any other sideeffects?

Reply:
ah, you are right, the mock was just not expecting the call. one should not atttempt to write confusing ternary operators when half falling asleep after a conference :-D